pkg/features/kube_features.go

```diff
@@ -939,7 +940,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	LocalStorageCapacityIsolation: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.27
 
-	LocalStorageCapacityIsolationFSQuotaMonitoring: {Default: false, PreRelease: featuregate.Alpha},
+	LocalStorageCapacityIsolationFSQuotaMonitoring: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
is the isue fix in this pr: https://github.com/kubernetes/kubernetes/pull/112625? Is this PR just WIP than?

Reply:
I think this should be pending until #112625 is merged to fix the regression.

I still needs more verifying for the fix.

Reply:
Since the fsquota related bug is fixed, I think we can promote it again. I will update the KEP as soon as possible in https://github.com/kubernetes/enhancements/pull/3821